qml/BrowserWindow.qml

```diff
@@ -23,14 +23,14 @@ import QtQuick.Controls 1.1
 import QtQuick.Controls.Styles 1.1
 import LunaNext.Common 0.1
 import LuneOS.Service 1.0
-import LuneOS.Application 1.0
+import LuneOS.Application 1.0 as LuneOS
 import LuneOS.Components 1.0
 import "js/util.js" as EnyoUtils
 
-ApplicationWindow {
+LuneOS.ApplicationWindow {
     id: appWindow
 
-    type: WindowType.Card
+    type: LuneOS.ApplicationWindow.Card
```

Comment:
I never found a way to make it work for both Dekstop debug and on-device; does that solve it?
Though if that works on device, it's fine for me.